api.yaml

```diff
@@ -381,6 +381,65 @@ paths:
               #! end !#
               "bodyPassthrough": true
             }
+  /v1/cloud/resid/{resid}:
```

Comment:
nit/opinion: From what I can tell we don't usually abbreviate our endpoints. I think it would be more clear to use something like:  `/v1/cloud/resourceid/{resourceid}`

Reply:
fixed